compressors.lib

```diff
@@ -741,64 +744,69 @@ declare limiter_basic_mono author "Dario Sanfilippo";
 declare limiter_basic_mono copyright "Copyright (C) 2020 Dario Sanfilippo 
       <sanfilippo.dario@gmail.com>";
 declare limiter_basic_mono license "GPLv3 license";
-limiter_basic_mono(threshold, lag, attack, hold, release, x) = 
+limiter_basic_mono(lag, threshold, attack, hold, release, x) = 
```

Comment:
Since you have defined **limiter_basic_N** later on, it seems the **limiter_basic_mono** then could be defined as a specialisation of it ?  And a **limiter_basic_stereo** version possibly be added explicitly using the same technique?